recipes/libattr/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
```

Comment:
The `cmake_find_package_multi` generator is not used (no `find_package` in the `CMakeLists.txt`)

Using/testing `find_package` would be out of place, since `libattr` does not provide/install cmake scripts.


```suggestion
    generators = "cmake"
```

Reply:
@dotChris90 please, remove cmake_find_package_multi and we are good to go.

Reply:
@uilianries  my bad - i just looked the main PR comments ....